src/docs/auth-doc.json

```diff
@@ -0,0 +1,183 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Students Teach Auth Documentation",
+    "version": "1.0.0",
+    "title": "Students Teach",
+    "contact": {
+      "email": "emirsagit@gmail.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "localhost:3000",
+  "basePath": "/auth",
+  "tags": [
+    {
+      "name": "student",
+      "externalDocs": {
+        "description": "More about google signin",
+        "url": "https://developers.google.com/identity/sign-in/web/sign-in"
+      }
+    }
+  ],
+  "schemes": ["https", "http"],
+  "paths": {
+    "/signin": {
+      "post": {
+        "tags": ["student"],
+        "summary": "Logs student into the system",
+        "operationId": "loginUser",
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Student object needs to be added to Students",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Signin"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "JWT token includes user informations"
+          },
+          "400": {
+            "description": "wrong email or password"
+          }
+        }
+      }
+    },
+    "/signup": {
+      "post": {
+        "tags": ["student"],
+        "summary": "Create a new student",
+        "description": "",
+        "operationId": "signupStudent",
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Student object needs to be added to Students",
```

Comment:
What is students at the end? if it's students collection you can type collection so it's more understandable